test/test_utils.ts

```diff
@@ -1,21 +1,38 @@
 import sinon from "sinon";
 import { expect } from "chai";
 
+class Headers extends Map {
```

Comment:
Is there a reason you didn't use the existing `Headers` global from [here](https://github.com/Kinto/kinto-http.js/blob/master/test/setup-globals.ts#L6)?